Content.Server/GameTicking/Rules/SecretRuleSystem.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using System.Linq;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.GameTicking.Rules;
+
+public class SecretRuleSystem : GameRuleSystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly GameTicker _ticker = default!;
+
+    public override string Prototype => "Secret";
+
+    private readonly IReadOnlyDictionary<string, float> _ruleTable = new Dictionary<string, float>
+    {
+        { "Traitor", 0.75f },
+    };
+
+    private readonly List<string> _attachedRules = new();
```

Comment:
does this need to be a list? looks like it only selects one rule

Reply:
instinctual future proofing moment
I'll make it a singular string.